homeassistant/components/esphome/__init__.py

```diff
@@ -256,13 +256,16 @@ async def try_connect(tries: int = 0, is_disconnect: bool = True) -> None:
             # really short reconnect interval.
             tries = min(tries, 10)  # prevent OverflowError
             wait_time = int(round(min(1.8 ** tries, 60.0)))
-            _LOGGER.info("Trying to reconnect to %s in %s seconds", host, wait_time)
+            if tries == 1:
+                _LOGGER.info("Trying to reconnect to %s in the background", host)
+            _LOGGER.debug("Retrying %s in %d seconds", host, wait_time)
             await asyncio.sleep(wait_time)
 
         try:
             await cli.connect(on_stop=try_connect, login=True)
         except APIConnectionError as error:
-            _LOGGER.info(
+            logger = _LOGGER.info if tries == 0 else _LOGGER.debug
```

Comment:
We can also use `_LOGGER.log(level, message)`.

Reply:
Thank you, I prefer that.

Reply:
Oh, sorry, I now see that our reviews passed each other. 😞

Reply:
No worries. :smile: